### DISCUSSION TITLE
Rename `infra` to `platform` or `infrastructure`

### DISCUSSION THREAD

**Discussion by JayDoubleu** [Azure/azure-dev#2309] [Ideas]
It appears that `infra` is the only reference in schema that's shorthand.
I believe it would be better to call it by more suitable name like `infrastructure` or `platform` given the impact this project is going to have in the future.

**Comment by savannahostrowski** [comment]
To be clear, you'd like to see the `azure.yaml` schema property `infra` renamed to be `infrastructure` or `platform` and are not asking to change the name of the directory we look for IaC files in right? The latter is configurable in the `azure.yaml`.

That said, I think that `infra` communicates the same intent as `infrastructure` but it's just a bit shorter. Personal preference probably comes into play here!

**Reply by JayDoubleu** [reply]
Thanks for the response, @savannahostrowski . I'm a huge fan, by the way.

You're absolutely right; this is 100% personal preference on my part. Hence, I thought to open this discussion rather than an issue. To be honest, I'm not fond of the term **"infrastructure"**. It reminds me of those days when **Devs** would call on the **Infra** guys to accomplish **X** or grant them permission to do **Y**, before the advent of DevOps and Platform Engineering.

You can probably judge from my response which side of the wall I'm coming from. Looking at `azd` from a certain point of view, I see a potential orchestrator or a thin wrapper around `terraform` and `bicep` to keep things **"DRY"** similar to `terragrunt` but azure native. 

Given that the **"services"** are being deployed onto the **"platform"** It might just make sense to take in into the consideration for the future schema changes :)
